dragon.go

```diff
@@ -43,8 +46,13 @@ func Imports() error {
 	if err != nil {
 		return err
 	}
+	err = tmp.Sync()
+	if err != nil {
+		return err
+	}
+	tmp.Close()
```

Comment:
Is this line unnecessary?
I think `file already closed` happen at [defer](https://github.com/monochromegane/dragon-imports/pull/5/files#diff-88f0d315e2b9fb003685d4b18b9f8fccR24) .

Reply:
Thanks! I understood that necessity.